modules/ROOT/pages/getting-started.adoc

```diff
@@ -11,61 +11,73 @@ NOTE: For more information on configuration, refer to the documentation for xref
+. Launch the VM using `aws ec2 run-instances`. The Ignition file can be passed to the VM as its https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html#instancedata-add-user-data[user data]. You can skip this step if you just want SSH access; simply pass a https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html[registered key name] via `--key-name`. This provides an easy way to test out FCOS without first creating an Ignition configuration.
 
-.Example launching Fedora CoreOS on AWS using an Ignition configuration file
+.Example launching FCOS on AWS using an Ignition configuration file
 [source, bash]
 ----
 aws ec2 run-instances <other options> --image-id <ami> --user-data file://example.ign
 ----
 
-.Example launching Fedora CoreOS on AWS using a registered SSH key pair
+.Example launching FCOS on AWS using a registered SSH key pair
 [source, bash]
 ----
 aws ec2 run-instances <other options> --image-id <ami> --key-name my-key
 ----
 
 While the AWS documentation mentions cloud-init and scripts, FCOS does not support cloud-init or the ability to run scripts from user-data. It accepts only Ignition configuration files.
 
-=== Launching with QEMU
-. Go to the https://getfedora.org/coreos/download/[download page] to retrieve the latest image suitable for QEMU.
+=== Launching with QEMU or libvirt
+. https://getfedora.org/coreos/download/[Download and verify] the latest image suitable for QEMU.
 
 . Uncompress the file. Assuming that the downloaded file is called `fedora-coreos-qemu.qcow2.xz`, the command would be the following:
 +
-`unxz fedora-coreos-qemu.qcow2.xz`
-+
-. Ensure the correct ownership of the file:
-+
-`chown core:kvm fedora-coreos-qemu.qcow2`
+[source, bash]
+----
+unxz fedora-coreos-qemu.qcow2.xz
+----
 +
-TIP: The above example assumes that you have created the user named `core` on your local machine. If you did not, replace `core` with the user that will boot FCOS on the local host.
+TIP: Make sure that your user has access to `/dev/kvm` (this should already be the case on modern distros), or if `/dev/kvm` is owned by the `kvm` group, adding yourself to that group with `usermod -aG kvm $USER`.
 
-. Specify the Ignition file to the hypervisor. In the case of QEMU, this is done with the `-fw_cfg` parameter, which sets the `opt/com.coreos/config` key in the QEMU firmware configuration device.
+. Launch the VM using `qemu-kvm` or `virt-install`. The Ignition file is passed to the VM via the `-fw_cfg` parameter, which sets the `opt/com.coreos/config` key in the QEMU firmware configuration device. For libvirt, you must pass this through the `--qemu-commandline` argument.
 +
-You must use absolute file paths here. QEMU cannot resolve relative file paths for these options.
+.Example launching FCOS with QEMU (no persistent storage)
```

Comment:
is there a better way to say this? `no persistent storage` is misleading in this context IMHO. When I hear "no persistent storage" I think of a live/ephemeral system, which is not the case here. If I reboot my changes will still be there.

This is more of a "create snapshot and boot into snapshot" than it is "no persistent storage".

Reply:
I tweaked this bit!

Reply:
Unfortunately I still think *temporary* is the wrong term. It's not actually temporary either. Maybe we should concentrate on the "snapshot" terminology. I'll grab you in IRC.

Reply:
ok, I was a bit mistaken here as I thought this was the heading for the `virt-install --disk backing_store=` invocation. This is resolved.